crypto/secp256k1/secp256k1.go

```diff
@@ -190,38 +251,35 @@ func (pubKey PubKey) Type() string {
 // VerifySignature verifies a signature of the form R || S.
 // It rejects signatures which are not in lower-S form.
 func (pubKey PubKey) VerifySignature(msg []byte, sigStr []byte) bool {
-	if len(sigStr) != 64 {
+	// if len(sigStr) != 64 {
```

Comment:
Do we need the comments within this function?

Reply:
(see my comment above) This was forward-ported from old peppermint. Feel free to remove